bot/exts/fun/fun.py

```diff
@@ -60,13 +61,14 @@ async def roll(self, ctx: Context, num_rolls: int = 1) -> None:
             raise BadArgument(f"`{Client.prefix}roll` only supports between 1 and 6 rolls.")
 
     @commands.command(name="randomcase", aliases=("rcase", "randomcaps", "rcaps",))
-    async def randomcase_command(self, ctx: Context, *, text: clean_content(fix_channel_mentions=True)) -> None:
+    async def randomcase_command(self, ctx: Context, *, text: str | None) -> None:
         """Randomly converts the casing of a given `text`."""
```

Comment:
Could you update the docstring to say the argument can be text, a reply, or a message link?

Reply:
was meant to be a request changes as would like this to be done ^